code/game/objects/items/stacks/rods.dm

```diff
@@ -3,6 +3,7 @@
 	desc = "Some rods. Can be used for building, or something."
 	singular_name = "metal rod"
 	icon_state = "rods"
+	item_state = "rods"

```

Comment:
will need icon_override on this one and to put the the sprites for it at the right location; it updates in hand too!